drivers/sensor/lsm6dsl/lsm6dsl.c

```diff
@@ -0,0 +1,447 @@
+/* lsm6dsl.c - Driver for LSM6DSL accelerometer, gyroscope and
+ * temperature sensor
+ */
+
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sensor.h>
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <misc/byteorder.h>
+#include <misc/__assert.h>
+
+#include "lsm6dsl.h"
+
+static inline int lsm6dsl_reboot(struct device *dev)
+{
+	struct lsm6dsl_data *data = dev->driver_data;
+	const struct lsm6dsl_config *config = dev->config->config_info;
+
+	if (i2c_reg_update_byte(data->i2c_master, config->i2c_slave_addr,
+				LSM6DSL_REG_CTRL3_C,
+				LSM6DSL_MASK_CTRL3_C_BOOT,
+				1 << LSM6DSL_SHIFT_CTRL3_C_BOOT) < 0) {
+		return -EIO;
+	}
+
+	k_busy_wait(50 * USEC_PER_MSEC);
```

Comment:
Could you please add a comment to indicate this is the turn on time according to the datasheet?